hw03_murphya.py

```diff
@@ -0,0 +1,90 @@
+# Author: Aaliyah Murphy
+# Username: Murphya
+#
+# Assignment: Hw03
+# Google Doc Link:https://docs.google.com/document/d/1QOq5mSUdxNJacGKRdWqrQEk5mgNh5cuqwZO52XxC4JY/edit?usp=sharing
+#
+
+import turtle
+
+wn = turtle.Screen()
+abby = turtle.Screen()
+abby.bgpic('skyy.gif')
+
+abby = turtle.Turtle()
+aaliyah = turtle.Turtle()
+
+#This code makes the horizon line and makes the water blue.
+abby.color("#5980C3", "#5980C3")
+abby.begin_fill()
+abby.penup()
+abby.right(90)
+abby.forward(65)
+abby.pendown()
+abby.left(90)
+abby.forward(375)
+abby.left(180)
+abby.forward(700)
+abby.left(90)
+abby.forward(200)
+abby.left(90)
+abby.forward(700)
+abby.left(90)
+abby.forward(200)
+abby.left(90)
+abby.end_fill()
+
+abby.penup()
+abby.goto(-40, -65)
+abby.pendown()
+#This code makes the ship dock
+abby.forward(250)
+abby.left(90)
+abby.color("#8E5B3A", "#8E5B3A")
```

Comment:
All of this should be at the beginning of `main()`. There should be no code at the highest indentation level. 

If you are having scoping issues, then you probably need to pass a turtle into functions:

```
def draw_something(turt):
    turt.forward(10)
    ...

def main():
    t = turtle.Turtle()
    draw_something(t)
```